### PR TITLE
feat: Rework of tokens (dark mode foundation)

### PR DESCRIPTION
**Pull request by magnuh**
Source of truth: https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8

### CODE REVIEW

**Review by felicia-haggqvist — COMMENTED** [review]
Noticed this when testing the vue-repo with these css-changes that does not look like this before these css-changes: 
Skärmavbild 2024-03-04 kl  15 13 24
<br>
Skärmavbild 2024-03-04 kl  15 13 14
<br>
Skärmavbild 2024-03-04 kl  15 13 05
<br>
Skärmavbild 2024-03-04 kl  15 11 12

**Comment by magnuh** [comment]
> Noticed that the background-color and text-color have changed when testing the vue-repo with these css-changes 

Did you include the new brand.css files with the new tokens?

**Comment by felicia-haggqvist** [comment]
> > Noticed that the background-color and text-color have changed when testing the vue-repo with these css-changes
> 
> Did you include the new brand.css files with the new tokens?

Whoops! No! 🙈 😄

**Review by felicia-haggqvist — APPROVED** [review]

File: tokens/finn.no/deprecated.yml [hunk 1]
```diff
@@ -530,12 +530,28 @@ s:
         default: blue-600
       positive:
         default: green-50
+        selected:
+          _: green-50
+          hover: green-100
+          active: green-200
```
Comment (BalbinaK) [inline tokens/finn.no/deprecated.yml:536]: The document you linked to seems to have been updated and those tokens are not needed:
https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831&range=68:70

**Review by BalbinaK — APPROVED** [review]
LGTM!

File: component-classes/index.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const box = {
   box: 'group block relative break-words last-child:mb-0 p-16 rounded-8', // Relative here enables w-clickable
   bleed: '-mx-16 sm:mx-0 rounded-l-0 rounded-r-0 sm:rounded-8', // We target L and R to override the default rounded-8
   info: 's-bg-info-subtle',
-  neutral: 's-bg-subtle',
+  neutral: 's-bg-sunken',
```
Comment (BalbinaK) [inline component-classes/index.js:56]: I tried testing these changes in React by linking them there + copying over missing tokens, but the `--w-s-color-background-sunken` token that this class was looking for was missing.
Screenshot 2024-03-07 at 10 42 58

Comment (BalbinaK) [inline component-classes/index.js:56]: I tried testing these changes in React by linking them there + copying over missing tokens, but the `--w-s-color-background-sunken` token that this class was looking for was missing.
Screenshot 2024-03-07 at 10 42 58

**Review by BalbinaK — CHANGES_REQUESTED** [review]
I tried testing these changes in React by linking them there + copying over missing tokens, but the `--w-s-color-background-sunken` token that this class was looking for was missing.
Screenshot 2024-03-07 at 10 42 58

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**